crates/solvers/openapi.yml

```diff
@@ -311,7 +312,71 @@ components:
           type: boolean
         class:
           $ref: "#/components/schemas/OrderClass"
-
+        protocolFees:
+          description: |
+            Any protocol fee policies that apply to the order.
+          type: array
+          items:
+            $ref: "#/components/schemas/FeePolicy"
+    FeePolicy:
+      description: |
+        A fee policy that applies to an order.
+      type: object
+      oneOf:
+        - $ref: "#/components/schemas/SurplusFee"
+        - $ref: "#/components/schemas/PriceImprovement"
+        - $ref: "#/components/schemas/VolumeFee"
+    SurplusFee:
+      description: |
+        If the order receives more than limit price, pay the protocol a factor of the difference.
+      type: object
+      properties:
+        kind:
+          type: string
+          enum: ["surplus"]
+        maxVolumeFactor:
+          description: Never charge more than that percentage of the order volume.
+          type: number
+          example: 0.1
```

Comment:
How about specifying the limits like it is proposed in the following PR?
https://github.com/cowprotocol/services/pull/2538/files#diff-ea5f6887f8360abaf702f3dcece07c3b2e87022458254f48bb2fb66adf15b95fR1528-R1535